### PR TITLE
Add coveralls and fix readme badges

### PR DESCRIPTION
**Pull request by joshblum**
Just a suggestion, the coveralls badge is nice to have to keep track of test coverage however and also I feel it gives people more confidence when choosing a project if the coverage is high.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,8 +1,13 @@
 .. image::  https://raw.githubusercontent.com/jgorset/django-kronos/master/docs/banner.png
 
-.. image:: https://api.travis-ci.org/jgorset/django-kronos.svg?branch=master
+.. image:: https://coveralls.io/repos/github/joshblum/django-kronos/badge.svg?branch=master
+    :target: https://coveralls.io/github/joshblum/django-kronos?branch=master
```
Comment (joshblum) [inline README.rst]: @jgorset If you want this you can register (free) at https://coveralls.io/ and just change the links here

Comment (joshblum) [inline README.rst]: It appears this is necessary for the tests to pass since Travis looks for the correct coveralls repo!

**Comment by joshblum** [comment]
Test's can pass if you setup a coveralls account since it's integrated with Travis! Let me know what you think :)

**Comment by jgorset** [comment]
Totally! Thanks @joshblum! :sparkles: :cake: :sparkles:

**Pull request merged**